exercises/queen-attack/.meta/template.j2

```diff
@@ -0,0 +1,38 @@
+{%- import "generator_macros.j2" as macros with context -%}
+{{ macros.header(imports=['Queen']) }}
+
+class {{ exercise | camel_case }}Test(unittest.TestCase):
+{% set suite_tests = cases -%}
+
+{% for cases in suite_tests -%}
+    # {{ cases["description"] }}
+    {% for case in cases["cases"] -%}
+    def test_{{ case["description"] | to_snake }}(self):
+        {% if case["property"] == "canAttack" -%}
+        {% set white_queen = case["input"]["white_queen"]["position"] -%}
+        {% set black_queen = case["input"]["black_queen"]["position"] -%}
+        self.assertIs(Queen({{ white_queen["row"] }}, {{ white_queen["column"] }}).can_attack(Queen({{ black_queen["row"] }}, {{ black_queen["column"] }})), {{ case["expected"] }})
+        {% else -%}
+        {% set queen = case["input"]["queen"]["position"] -%}
+        {% if case["expected"] == 0 -%}
```

Comment:
```suggestion
        {% if not case is error_case -%}
```

Reply:
Bump.

Reply:
verify if `expected` key is a int or dict was a headache :/, the function `error_case` made it easier :D